lib/plugins/aws/package/compile/events/websockets/lib/api.js

```diff
@@ -17,6 +17,18 @@ module.exports = {
       this.serverless.service.provider.websocketsApiRouteSelectionExpression ||
       '$request.body.action';
 
+    // Tags
+    const tagsMerged = Object.assign(
+      {},
+      this.serverless.service.provider.stackTags,
```

Comment:
We should not merge `stackTags` here - they will be appended by CloudFormation automatically for all eligible resources during CloudFormation deployment

Reply:
For my test case `stackTags` are not applied (it has been added to DDB, Lambda etc.). This is where my search began.

Only after adding the changes to the code here did they appear in the AWS console.

Reply:
It might be an issue with CloudFormation's application of `stackTags` for that specific resournce then - still, I would not merge them because `stackTags`'s intention is to be passed to CloudFormation update/create operations, not directly to resources defined in CloudFormation template.

Reply:
Changed - we shouldn't fix CloudFormation's issues.